src/form.js

```diff
@@ -95,9 +95,12 @@ export default function form({fields, validate = () => ({})}) {
         forceValidate: () => this.touch(fields),
         values: () => this.state.values,
         onValues: values => {
-          if (!this.props.value) {
-            this.setState({ values });
+          this.setValues(values);
+          if (typeof values === 'object') {
```

Comment:
Maybe instead of this, we forceValidate is changed  to accept an array of fields, but default to all the fields:
`forceValidate: (fieldsToValidate = fields) => this.touch(fieldsToValidate),`.

With this the user can update the values of the fields without being forced to mark them as touched.  If the user wants both, they can just call 
```
onValues(changes);
forceValidate(Object.keys(changes));
```
themselves.  This seems like it would be more flexible to me, but I would like to hear your opinion!